overlay/base/usr/lib/system/_os/podman.py

```
@@ -82,6 +82,27 @@ def podman(
+    ).strip()
+
+
+def storage_run_root() -> str:
+    return subprocess.check_output(
+        podman_cmd("info", "--format", "{{.Store.RunRoot}}"),
+        text=True,
+    ).strip()
+
+
+def storage_graph_driver() -> str:
+    return subprocess.check_output(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)